src/app/layout.tsx

```diff
@@ -1,44 +1,27 @@
-import type { Metadata } from "next";
 import { MantineProvider } from "@mantine/core";
-import localFont from "next/font/local";
+import Header from "../components/AppShell/Header";
 import "@mantine/core/styles.css";
 import "./globals.css";
 
-const geistSans = localFont({
-    src: "./fonts/GeistVF.woff",
-    variable: "--font-geist-sans",
-    weight: "100 900",
-});
-const geistMono = localFont({
-    src: "./fonts/GeistMonoVF.woff",
-    variable: "--font-geist-mono",
-    weight: "100 900",
-});
-
-export const metadata: Metadata = {
-    title: "Create Next App",
-    description: "Generated by create next app",
-};
-
 export default function RootLayout({
     children,
 }: Readonly<{
     children: React.ReactNode;
 }>) {
     return (
         <html lang="en">
-            <body
-                className={`${geistSans.variable} ${geistMono.variable} antialiased`}
-            >
+            <body className="pt-14 antialiased">
+                <div className="fixed inset-0 z-0 bg-neutral-200" />
```

Comment:
Instead of adding a div as a background, consider just making body `fixed inset-0 overflow-scroll-y` since it is very uncommon to elevate all elements with z-level

Reply:
apply  ```fixed inset-0 overflow-y-scroll``` on body make the whole page unable to scroll.

Reply:
I just tested it out, you were right, it seems that CSS has some issues dealing with overflow on fixed objects. I tested an alternative way where only the children were wrapped in a div with h-[{screen - header height}], and overflow was applied on that div instead, and it worked. I think you should try that instead.

Reply:
The issue with the current code is that a scrollbar piercing the header appears, and it is rather unintuitive from the user's perspective to see a scrollbar on top of a fixed element.